packages/ark/src/lib/angular/async-context.ts

```diff
@@ -5,7 +5,7 @@ export class AsyncContext<T> {
     return this._value;
   }
 
-  update(value: T): void {
+  update(value: T | undefined): void {
```

Comment:
because its a publishable library we need to support strict mode and make sure null & undefined are explicitly set
```suggestion
  update(value?: T | null): void {
```